op-enable.go

```diff
@@ -380,3 +357,22 @@ func getArgs(s DockerHandlerSettings, dd driver.DistroDriver) string {
 
 	return strings.Join(args, " ")
 }
+
+
+func getDockerUrls(env string) (string, string) {
```

Comment:
if you're wondering how did I come up with my comment above, just look at `func getDockerUrls(env string) (string, string)` method as a different person and try to understand what it's telling you. (it did not make much sense to me and therefore I had to read code and saw all this custom logic).